tendrl/ceph_integration/manager/crud.py

```diff
@@ -119,3 +123,14 @@ def delete(self, object_type, object_id):
             )
         else:
             raise NotImplementedError(object_type)
+
+    def sync_request_status(self, request):
+        count = 0
+        while count < 30:
+            gevent.sleep(10)
+            if request.state == "complete" and not request.error:
```

Comment:
when ceph timeout the request it gives response like {'status': 1, 'err': 'rbd command timed out', 'out': ''}
i feel this if condition will omit this response.